css/style.css

```diff
@@ -15,48 +17,59 @@ body, input {
   margin: 20px;
   border-radius: 30px;
   background-color: #ccc;
-  color: black; }
+  color: black !important; 
+}
 
 #githubHeaderTag {
   position: absolute;
   padding: 10px;
-  margin: 20px; }
+  margin: 20px; 
+}
 
 #toggle-day {
-  display: none; }
+  display: none; 
+}
 
 #toggle-night,
 #toggle-day {
-  cursor: pointer; }
+  transition: 1s;
+  cursor: pointer; 
+}
 
 #header-div {
   text-align: center;
-  padding: 20px; }
+  padding: 20px; 
+}
 
 #header-div > a > img {
   height: 150px;
-  width: 150px; }
+  width: 150px; 
+}
 
 #header-div-a:link,
 #header-div-a:active,
 #header-div-a:visited {
   text-decoration: none;
   color: black;
   font-family: Samarkan;
-  font-size: 1.25em; }
+  font-size: 1.25em; 
```

Comment:
If you want to use em then use in in every font-size styles otherwise revert to px.